### DISCUSSION TITLE
Confused by keyboard layouts

### DISCUSSION THREAD

**Discussion by glow12121** [labwc/labwc#1227] [Q&A] [answered]
I've noticed that LabWC and my main desktop environment have slightly different keyboard layouts.

For example, in Gnome, Shift + tilda type `~`.

But when I launch `labwc -s gnome-terminal` and type the same there, instead of `~` I get a fraction: `1/2`. A few more keys are in wrong places. Why?

**Comment by Consolatis** [comment, marked as answer]
Your gnome desktop might use a slightly different variant of the main layout compared to labwc.
In labwc, you can configure your keyboard layout (and its variant) in `~/.config/labwc/environment` as shown [here](https://github.com/labwc/labwc/blob/master/docs/environment).
